simcase-game/src/components/Header.js

```diff
@@ -3,11 +3,10 @@ import React from "react";
 import { jsx } from "@emotion/core";
 import { createPortal } from "react-dom";
 import { Dialog } from "@reach/dialog";
-import "@reach/dialog/styles.css";
-import Role from "../views/Role";
-import Leaderboard from "../views/Leaderboard";
-import Walkthrough from "../views/Walkthrough";
 import { navigate } from "@reach/router";
+import MyModal from "../views/Modal";
+
+import "@reach/dialog/styles.css";
```

Comment:
Is necessary to import this file to add styles to your modal?

Reply:
The css defines the styles for the dialog modal. It pertains only to the app menu.